regimes/co/extensions.go

```diff
@@ -91,4 +92,41 @@ var extensionKeys = []*cbc.KeyDefinition{
 			},
 		},
 	},
+	{
+		Key: ExtKeyDIANCorrectionDebit,
+		Name: i18n.String{
+			i18n.EN: "DIAN correction code for debit notes",
+			i18n.ES: "Código de corrección DIAN para notas de débito",
+		},
+		Codes: []*cbc.CodeDefinition{
+			{
+				Code: "1",
+				Name: i18n.String{
+					i18n.EN: "Interest",
+					i18n.ES: "Intereses",
+				},
+			},
+			{
+				Code: "2",
+				Name: i18n.String{
+					i18n.EN: "Pending charges",
+					i18n.ES: "Gastos por cobrar",
```

Comment:
The Spanish names are those in the DIAN docs. I'm not 100% sure about the English translation in this particular case.

Reply:
Translation seems good to me.